projects/self-creation-portal/src/app/components/resource-holder/resource-holder.component.html

```diff
@@ -12,14 +12,16 @@
    </div>
     @for (list of filters.filteredLists; track list) {
         <div class="pb-1">
-        <lib-card [list]="list" [showActionButton]="filters.showActionButton" (buttonClickEvent)="handleButtonClick($event)"></lib-card>
+        <lib-card [list]="list" [showActionButton]="filters.showActionButton" (buttonClickEvent)="cardButtonClick($event)"></lib-card>
         </div>
     } 
     } 
     @else 
     {
         <div class="flex flex-col  justify-center items-center text-center">
-            <lib-no-result-found imagePath="assets/images/empty-state.png" [noResultMessage]="noResultMessage"></lib-no-result-found>
+            @if(filters.filteredLists.length === 0 && isDataLoaded){
```

Comment:
why we need isDataLoaded ?

Reply:
to check the api response received or not so to show nodata found ui once api response occurs only.